src/transformers/models/perceiver/modeling_perceiver.py

```diff
@@ -823,7 +823,7 @@ def forward(
         ...     project_pos_dim=256,
         ...     trainable_position_encoding_kwargs=dict(
         ...         num_channels=256,
-        ...         index_dims=config.image_size**2,
+        ...         index_dims=config.image_size ** 2,
```

Comment:
Let's leave this as is, it's not linked to this PR.

Reply:
Reverted, thanks (think that somehow came with one of the `make` scripts, but my env may not've been fully properly setup)